Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Use the official Ubuntu 24.04 image as the base image
+FROM ubuntu:24.04
```

Comment:
We don't need a full ubuntu image its larger then we need.

Reply:
Thanks, I replaced it with minimal image.